### PR TITLE
Redux Counter - Rane Wallin

### PR DESCRIPTION
**Pull request by RaneWallin**

### CODE REVIEW

**Comment by RaneWallin** [comment]
Stretch and MVP complete

File: src/components/Counter.js [hunk 1]
```diff
@@ -6,11 +6,13 @@ class Counter extends Component {
     incrementIfOdd = () => {
         // Stretch Problem: Implement an increment function that
         // only increments if the counter value is odd
+        if(this.props.count % 2 != 0) this.props.increment();
```
Comment (GannonDetroit) [inline src/components/Counter.js]: good job

File: src/components/Counter.js [hunk 2]
```diff
@@ -20,20 +22,19 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => {/* Fill me in */ }}>
+                <button onClick={() => { this.props.increment() }}>
```
Comment (GannonDetroit) [inline src/components/Counter.js]: good job

File: src/reducers/index.js [hunk 3]
```diff
@@ -13,8 +13,10 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+          return { count: state.count + 1 }
```
Comment (GannonDetroit) [inline src/reducers/index.js]: good job

**Pull request merged**